src/core/components/NavBar.tsx

```diff
@@ -6,12 +7,30 @@ import { isInIframe } from '../utils/isInIframe';
 
 const FEAT_UPDATED_DESIGN = import.meta.env.VITE_FEAT_UPDATED_DESIGN;
 
+interface ModelOption {
+  id: string;
+  name: string;
+  icon: JSX.Element;
+}
+
+const modelOptions: ModelOption[] = [
+  { id: 'deepseek', name: 'DeepSeek RI 67TB', icon: <DeepseekIcon /> },
+  { id: 'kavaai', name: 'Kava AI', icon: <>Logo coming soon</> },
+];
+
 const NavBar = () => {
   const isIframe = isInIframe();
   const showNavBar = !isIframe && FEAT_UPDATED_DESIGN;
+  const [dropdownOpen, setDropdownOpen] = useState(false);
+  const [selectedModel, setSelectedModel] = useState(modelOptions[0]);
 
   if (!showNavBar) return null;
 
+  const handleSelect = (model: ModelOption) => {
+    setSelectedModel(model);
+    setDropdownOpen(false);
+  };
```

Comment:
Eventually, we will also be able to set the model name which we can use to key into the `MODEL_REGISTRY` to get system prompt, tools, components, etc.